spec/features/index_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+RSpec.describe "Ingredients", type: :feature do
+  describe "Ingredients Index" do
+    it "displays a list of all ingredients with their name and cost" do
+      ingredient1 = Ingredient.create(name: "Ground Beef", cost: 2)
+      ingredient2 = Ingredient.create(name: "Salt", cost: 4)
+
+      visit "/ingredients"
+
+      expect(page).to have_content("#{ingredient1.name}: #{ingredient1.cost}")
+      expect(page).to have_content("#{ingredient2.name}: #{ingredient2.cost}")
```

Comment:
Overall good feature test! I like that you tested exactly what you ended up writing in the view. It may be useful in the future to separate them out by attribute names, e.g.
```
Name: "#{ingredeint1.name}"
Cost: "#{ingredient1.cost}"
```
that way you know which element is which. What if an ingredient is called "$12 spaghetti" and it costs $12, or some weird combo like that?